src/index.ts

```diff
@@ -0,0 +1,11 @@
+export { default as Calendar } from './calendar';
```

Comment:
We need to export all the props as well

Reply:
Fixed. 
And I trigger a new snapshot build for this branch